client/galaxy/scripts/mvc/tool/tool-genomespace.js

```diff
@@ -0,0 +1,23 @@
+// Provides support for interacting with the GenomeSpace File Browser popup dialogue
+define([], function() {
+
+// tool form templates
+return {
+    openFileBrowser: function( options ) {
+        var GS_UPLOAD_URL = "https://gsui.genomespace.org/jsui/upload/loadUrlToGenomespace.html?getLocation=true";
```

Comment:
Could this location be made centrally configurable somehow?  If so, that would be pretty useful since it could be made configurable at install time to point to the Australian mirror for example.